plugins/languages/treesitter/treesitter.nix

```diff
@@ -118,7 +118,11 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
       keymaps =
         let
-          mkKeymap = default: helpers.defaultNullOpts.mkStr default "Key shortcut";
+          mkKeymap =
```

Comment:
I guess it's out of scope here, but if you used a genAttrs or a mapAttrs, you could also use the option's name in the description... Although that's not really important since that should be obvious.

```nix
keymaps =
  mapAttrs
  (
    name: default:
    helpers.defaultNullOpts.mkNullable (
      with helpers.nixvimTypes;
      oneOf [
        str
        bool
        rawLua
      ]
    ) default "Key shortcut used for `${name}`, or false to unset.";
  )
  {
    init_selection = "gnn";
    node_incremental = "grn";
    scope_incremental = "grc";
    node_decremental = "grm";
  };
```

Reply:
Yeah, I thought about whether or not it should be more descriptive, too. I didn't know if it was helpful since we are reading this description underneath each of the corresponding names options in the docs. But, I suppose you have to parse the final name when reading the giant config option path.